# Cordyceps_militaris-GSSM

- Brief Repository Description

This repository contains a genome-scale metabolic model **iPC1469** for _Cordyceps militaris_.

- Abstract:
_Cordyceps militaris_ is an industrially important fungus, which is often used in Asia as a traditional medicine. Up to now, there has been a published genome-scale metabolic model of _C. militaris_, iNR1329. Apart from the genomic and transcriptomic data, metabolome has a potential in addressing biological functions at systems level. To gain more insights into global metabolism of _C. militaris_, the metabolic footprinting of _C. militaris_ TBRC6039 was carried out by high-performance liquid chromatography coupled with electrospray ionization quadrupole time-of-flight mass spectrometry. Using various carbon sources, the differentially accumulated metabolites (DAMs) were significantly exhibited by pairwise comparison between the xylose- and glucose-grown cultures (C5 vs C6 cultures) followed with the sucrose- and glucose-grown cultures (C12 vs C6 cultures). Of the DAMs, the sphingolipid derivatives, including sphinganine, sphingosine, phytosphingosine, and ceramide, were significantly accumulated in the C5 culture when compared with other cultures. By incorporating the identified significant DAMs and transcriptome data, the enhanced GSMM of _C. militaris_ (iPC1469) was generated and efficiently used for probing a complexity in metabolic functions of _C. militaris_. The metabolic response of lipid metabolism in _C. militaris_ under different carbon sources, particularly sphingolipid biosynthesis, was explored. This study suggests that the sphingolipid biosynthetic capability in _C. militaris_ was dependent on carbon source for cell growth. This finding provides a comprehensive basis on the sphingolipid biosynthesis in _C. militaris_ that would be benefit to further redesigning its metabolic control for medicinal and functional food applications.

- Model KeyWords:
Sphingolipid, _Cordyceps militaris_, Genome-scale modeling, Metabolic footprinting, Systems biology
**GEM Category:** Species; **Utilisation:** Predictive simulation; **Field:** Genome-scale modeling; Metabolic
engineering; **Type of Model:** Reconstruction and refinement; **Taxonomy:** _Cordyceps_militaris_ ; **Metabolic System:** General Metabolism; **Condition:** Synthetic media;

- Reference: Genome-Scale Model Integration of Exometabolome and Transcriptome Unveils Carbon Assimilation Towards Sphin-golipid Biosynthetic Capability of _Cordyceps militaris_. Cheawchanlertfa P, Chitcharoen S, Raethong N, Liu Q, Chumnanpuen P, Soommat P, Song Y, Laoteng K and Vongsangnak W. (under review)

- Pubmed ID: 

- PMC ID: 

- Last update: 2022-06-09

- The model contains:

| Taxonomy | Model Name | Reactions | Metabolites| Genes |
| ------------- |:-------------:|:-------------:|:-------------:|-----:|
| _Cordyceps militaris_ | iPC1469 | 1,904 | 1,229 | 1,469 |

This repository is administered by Nachon Raethong ([@sysbiomics](https://github.com/sysbiomics))

## Installation

### Recommended Software:
* A functional Matlab installation (MATLAB 7.3 or higher).
* [RAVEN Toolbox 2.0](https://github.com/SysBioChalmers/RAVEN) for MATLAB (required for contributing to development). 
* libSBML MATLAB API ([version 5.16.0](https://sourceforge.net/projects/sbml/files/libsbml/5.13.0/stable/MATLAB%20interface/)  is recommended).
* [Gurobi Optimizer for MATLAB](http://www.gurobi.com/registration/download-reg).
* For contributing to development: a [git wrapper](https://github.com/manur/MATLAB-git) added to the search path.

### Installation Instructions
* Clone the [master](https://github.com/sysbiomics/Cordyceps_militaris-GSMM) branch from [sysbiomics GitHub](https://github.com//sysbiomics).

## Contributors
* Nachon Raethong (https://www.researchgate.net/profile/Nachon_Raethong) ([@nachonase](https://github.com/nachonase))
